broker/broker-app/src/main/resources/features.xml

```diff
@@ -151,7 +151,7 @@
     </feature>
 
     <feature name="broker-app" install="auto" version="${project.version}"
-             description="The Broker Application provides broker services for reliable message delivery in the DDF. ::Broker">
+             description="The Broker Application provides broker services for reliable message delivery.">
```

Comment:
We need to preserve the `::Broker` at the end since it is used as the application name in the UI